dotansimha/graphql-code-generator-community
# feat(react-query): support react query v5

## Description

Let TypeScript React-Query plugin support V5 payload syntax.

Related #318 

## Type of change

Please delete options that are not relevant.

      expected)

## How Has This Been Tested?

- Set `reactQueryVersion` to 5 in config.
- Run my [reproduce repo](https://github.com/neil585456525/graphql-codegen-plugin-react-query-v5-test) to check

## Checklist:

- [x] I have followed the
      [CONTRIBUTING](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) doc and the
      style guidelines of this project

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose
the solution you did and what alternatives you considered, etc...
